general/github/pull-requests/README.md

```diff
@@ -1,19 +1,23 @@
+6. After successful review, merge your branch into "staging". Make sure to delete your remote branch thereafter as to not clutter the number of branches on the repo.
 
 ## Peer Review Etiquette
 
-1. Check your ego at the door.  Peer reviews are an opportunity to learn and discuss code and should not be viewed in a personal manner.  Stay professional!
-2. Be mindful of other engineer's time.  While you're not expected to immediately drop everything you're doing to handle a code review, there is an expectation to handle them as soon as possible as to not hold up progress. In general, if you've received no feedback on a PR after 20 minutes, ping the code reviewers.
-3. As you're coding your changes, be mindful of scope creep.  While we definitely encourage leaving code better than you found it, often times it may be better to split off changes that don't belong with the task at hand.  The smaller your pull request, the easier the code review.
-4. Don't be afraid to include comments and provide explanations inside of your pull request!
-5. In the event that another programmer leaves a comment, make sure to reply in a timely manner and let the other programmer resolve the comment before merging.
-6. If another programmer approves the PR but a 2nd programmer makes a comment, be sure to address the comment before merging.
+1. Check your ego at the door. Peer reviews are an opportunity to learn and discuss code and should not be viewed in a personal manner. Stay professional!
+2. Be mindful of other engineer's time. While you're not expected to immediately drop everything you're doing to handle a code review, there is an expectation to handle them as soon as possible as to not hold up progress. In general, if you've received no feedback on a PR after 20 minutes, ping the code reviewers.
+3. Don't be afraid to include comments and provide explanations inside of your pull request!
+4. In the event that another programmer leaves a comment, make sure to reply in a timely manner and let the other programmer resolve the comment before merging.
+5. If another programmer approves the PR but a 2nd programmer makes a comment, be sure to address the comment before merging.
+
+## Best Practices
+
+1. Before opening a pull request, take the time to [review your changes](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-comparing-branches-in-pull-requests). This will help you write a more meaningful description, catch errors and other things (like commented out code) that should be addressed first, and give a sense of scale of the pull request.
+2. As you're coding and reviewing your changes, be mindful of scope creep. While we definitely encourage leaving code better than you found it, often times it may be better to split off changes that don't belong with the task at hand. The smaller your pull request, the easier the code review.
```

Comment:
I moved this from the etiquette section since it felt more related to best practices.